pandasai/__init__.py

```diff
@@ -642,6 +643,7 @@ def _get_environment(self) -> dict:
 
         return {
             "pd": pd,
+            "plt": matplotlib.pyplot,
```

Comment:
We don't need this, as we automatically import the relevant dependencies (with the right alias) with `import_dependency` a few lines below.

Reply:
> We don't need this, as we automatically import the relevant dependencies (with the right alias) with `import_dependency` a few lines below.

Roger that, with a next commit.

Reply:
Great, thanks a lot!